### PR TITLE
[YUNIKORN-134] Post 0.8.0 release - update roadmap.

### PR DESCRIPTION
**Pull request by yangwwei**
This PR updates the roadmap.

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -4,25 +4,27 @@
...
-* Workload simulator.
-* Prometheus integration (phase 2).
-* Grafana integration.
+* [YUNIKORN-1](https://issues.apache.org/jira/browse/YUNIKORN-1): Support app/task priority aware scheduling.
+* [YUNIKORN-2](https://issues.apache.org/jira/browse/YUNIKORN-2): Gang Scheduling.
+* [YUNIKORN-21](https://issues.apache.org/jira/browse/YUNIKORN-21): Optimize node sorting algorithms.
+* [YUNIKORN-42](https://issues.apache.org/jira/browse/YUNIKORN-42): High efficient scheduling events framework phase 1.
+* [YUNIKORN-33](https://issues.apache.org/jira/browse/YUNIKORN-33): Performance benchmark with Kubemark.
+* [YUNIKORN-131](https://issues.apache.org/jira/browse/YUNIKORN-131): Prometheus integration - phase 2.
+* [YUNIKORN-132](https://issues.apache.org/jira/browse/YUNIKORN-132): Publish helm charts for convenient access.
```
Comment (sunilgovind) [inline docs/roadmap.md:13]: Do we need this? 
Is there any public helm repo in apache where we can publish same?

File: docs/roadmap.md [hunk 2]
```diff
@@ -4,25 +4,27 @@
...
 
 **yunikorn-k8shim**
 
-* Security support.
-* Performance improvement: merging, packing requests.
-* 3rd Operator/controller integration, pluggable app discovery.
```
Comment (sunilgovind) [inline docs/roadmap.md:19]: How about detailed APP state management including FINISHED & FINISHING and other closing states?

File: docs/roadmap.md (hunk 1 above)
Comment (yangwwei) [reply docs/roadmap.md:13]: No, I don't think apache has that.
If you are not comfortable with this, I am fine to remove it.

Comment (wilfred-s) [reply docs/roadmap.md:13]: the helm chart could be deployed and accessible from github by using `chart-releaser` the index.yaml file we can even host on the standard yunikorn website but this is a lot of work and would need good amount of thought, planning and build changes too.

I would prefer to not do this for now

Comment (yangwwei) [reply docs/roadmap.md:13]: OK, let me remove it.

File: docs/roadmap.md (hunk 2 above)
Comment (yangwwei) [reply docs/roadmap.md:19]: That is more like a bug fix

File: docs/roadmap.md (hunk 1 above)
Comment (sunilgovind) [reply docs/roadmap.md]: Ok.

File: docs/roadmap.md (hunk 2 above)
Comment (sunilgovind) [reply docs/roadmap.md:19]: Basic APP state is there anyway, I am fine with this.

**Pull request closed without merging**